requirements-core.txt

```diff
@@ -3,6 +3,7 @@ defusedxml>=0.7.0
 lxml>=4.4.1
 matplotlib>=3.3.1
 networkx>=2.5
+nibabel==3.2.1
```

Comment:
Why the version is fixed?

Reply:
This version was written in the requirements of AC. But perhaps we can put a non-fixed version.